### DISCUSSION TITLE
add OPENAI_API_BASE_URL as a config PARAMETER so it can point to other OpenAI compatible proxy servers (I.e. local models)

### DISCUSSION THREAD

**Discussion by vrijsinghani** [ra-jeev/write-assist-ai#15] [Ideas]
By adding a config parameter to OpenAIConfig like OPENAI_API_BASE_URL so it can be used via LiteLLM, OLLAMA, etc...

**Comment by ra-jeev** [comment]
@vrijsinghani thank you for your request. I've a question, can this base_url work using the OpenAI package? I do see that OpenAI package supports adding a base_url configuration.

If yes, can you please create an issue so that I can add the support?
